graphql/mutation_resolver.go

```diff
@@ -581,18 +581,18 @@ func (r *mutationResolver) DetachProjectFromRepo(ctx context.Context, projectID
 	usr := mustHaveUser(ctx)
 	pRef, err := data.FindProjectById(projectID, false, false)
 	if err != nil {
-		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("error finding project %s: %s", projectID, err.Error()))
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("finding project %s: %s", projectID, err.Error()))
```

Comment:
this is probably `InternalServerError`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding project '%s': %s", projectID, err.Error()))
```